src/brevitas/graph/gpxq.py

```diff
@@ -35,7 +33,7 @@ class LayerHandler:
     forward_count: int = 0
 
 
-class gpxq_mode(ABC):
+class gpxq_mode(disable_enable_quantization):
```

Comment:
Maybe I would go for composability rather than inheritance.
Functionally might be the same, but intuitively I don't see why GPxQ should inherit from that